torch/csrc/jit/script/schema_matching.cpp

```diff
@@ -398,9 +399,24 @@ Value* emitBuiltinCall(
   if (!required) {
     return nullptr;
   }
-  if(variants.size() == 0) {
-    throw ErrorReport(loc) << "unknown builtin op";
+  // no operators found with the same name, print out similarly named operators
+  if (variants.size() == 0) {
+    auto close_operators = findSimilarOperators(name);
+    auto error = ErrorReport(loc);
+    auto user_function_name = name.toQualString();
+    error << "unknown builtin op: " << user_function_name << "\n";
+    if (close_operators.size() == 0) {
+      error << "Could not find any similar ops to " << user_function_name
+        << ". This op may not exist or may not be currently supported in TorchScript\n";
+    } else {
+      error << "Here are some suggestions: \n";
+      for (auto op: close_operators) {
+        error << "\t" << op.second.first.toQualString() << "\n";
```

Comment:
You're passing around a very complicated data structure (`std::multimap<int64_t, std::pair<Symbol, std::vector<std::shared_ptr<Operator>>>`) when you only need `std::vector<Symbol>`. I don't think it's good to spill the guts of your implementation out like that. 

If you want to return them in order, just use a `std::priority_queue` to maintain ordering and return the underlying container.